nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,53 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+//Respond with the json for all the meals
+router.get("/", async(request, response) => {
+    try {
+        response.json(meals);
+    } catch (error) {
+        throw error;
+    }
+});
+//Respond with the json for the meal with the corresponding id
+router.get("/:id", async(req, res) => {
+    try {
+        const { id } = req.params;
+        const mealWithId = meals.filter((meal) => meal.id === parseInt(id));
+        if (mealWithId < 1) {
```

Comment:
Operator '<' cannot be applied to types '{ id: number; title: string; maxNumberOfGuests: number; description: string; createdAt: string; price: number; }[]' and 'number'.ts(2365)